### ISSUE TITLE
[Feature Request] PWA (Progressive web app) support

### ISSUE THREAD

**Issue by edersong** [alexta69/metube#377]
It would be a nice feature to be able to install the website as an app on Android and iOS.
Ref.: https://blog.heroku.com/how-to-make-progressive-web-app

**Comment by PikuZheng** [comment]
chrome mobile works. why app？

**Comment by edersong** [comment]
Hello,
No an app, a Progressive Web App, some parameters where when we create a shortcut from MeTube at our mobile phone, it will work like an app.

**Comment by PikuZheng** [comment]
yes... pack a web to a app, cache front-end code inside the phone. hbuildx is a kind of pwa
Android can use shoutcut to create any custom shortcut
There are also frequent updates and upgrades. Overall, I don’t need to use it.
